data.json

```diff
@@ -765,12 +777,12 @@
     "ui": "web"
 },
 {
-    "name": "Vimium",
-    "id": "vimium",
-    "descr": "Plugin for Firefox, Pale Moon, Basilisk.",
-    "url": "https://addons.mozilla.org/en-US/firefox/addon/vimium/",
-    "license": "BSD-like",
-    "languages": [ "js probably" ],
+    "name": "Vimium-FF",
+    "id": "vimium-ff",
```

Comment:
The project is now called `Vimium-FF`, and no longer `Vimium` for Firefox. The old URL does not work: https://addons.mozilla.org/en-US/firefox/addon/vimium/

Do you mean it's still better to only change the `name` key, but keep the `id` key the same?

Reply:
> Do you mean it's still better to only change the `name` key, but keep the `id` key the same?

Yes, since it's the same project. Alternative would be to add it as a separate item stating that old one is abandoned and is now developed as Vimium-FF.

Reply:
Got it! Updated is pushed